static/docs/user-guide/introduction.md

```diff
@@ -0,0 +1,32 @@
+# Basic Concepts
+
+The DVC user guide constitutes DVC files and directories, DVC file format,
+external dependencies, external outputs, update a tracked file, anonymized usage
+analytics which are part of basic. Customizations has DVC autocomplete, plugins,
+developement version and contributing guide.
+
+Let's understand the basic concepts as you use dvc.
+
+## Workspace
+
+Work space is the current work directory.
+
+## Cache
+
+DVC cache is a hidden (by default it's located in the `.dvc/cache` directory)
+storage. For files that are under DVC control it keeps them and their different
+versions. You can find more information on cache as described in this [doc](https://dvc.org/doc/commands-reference/config#cache).
+
+## Remote
+
+In DVC you can have remote data storage or a remote git hosted code server.
```

Comment:
`remote git hosted code server` - I'm not sure I understand this. Where does it come from and what does it mean?

Reply:
Git can be hosted on any remote ip, if you put code + git on the remote ip, you could call the ip as remote git hosted code server (https://git-scm.com/book/en/v1/Git-on-the-Server, https://git-scm.com/book/en/v2/Git-on-the-Server-Setting-Up-the-Server). Please let me know after reading these you would prefer different wording.